astropy/utils/iers/iers.py

```diff
@@ -45,6 +52,11 @@
 IERS_B_URL = 'http://hpiers.obspm.fr/iers/eop/eopc04/eopc04_IAU2000.62-now'
 IERS_B_README = get_pkg_data_filename('data/ReadMe.eopc04_IAU2000')
 
+# LEAP SECONDS default file name, URL, and alternative format/URL
+IERS_LEAP_SECOND_FILE = get_pkg_data_filename('data/Leap_Second.dat')
+IERS_LEAP_SECOND_URL = 'https://hpiers.obspm.fr/iers/bul/bulc/Leap_Second.dat'
```

Comment:
Do we need mirrors?

Reply:
This is a good point - can we have `IERS_LEAP_SECOND_URL` and `IETF_LEAP_SECOND_URL` both be *lists* that are tried in order? I don't think we need to bake in a mirror right now, but I think the USNO experience suggests we should have *some* way to provide backup options...

Reply:
Note that they are mirrors of each other. I'd happily change to lists *after* @aarchiba's nice caching PR is merged, since then I don't have to do anything but put it in `sources`...

Reply:
So, concretely, let's postpone making this a list for a follow-up PR... I also want to collect system files on other types of operating systems.